src/nvim/eval.c

```diff
@@ -17625,7 +17625,7 @@ void ex_function(exarg_T *eap)
                                                (p[2] == 's'))))))
         skip_until = vim_strsave((char_u *)".");
 
-      /* Check for ":python <<EOF", ":tcl <<EOF", etc. */
+      /* Check for ":python <<EOF" */
```

Comment:
I think `:python` is the only such command currently available. I could preemptively put `:python3`, given `:lua` doesn't do anything.